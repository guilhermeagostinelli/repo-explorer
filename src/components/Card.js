import styled from "styled-components";

const Card = styled.div`
  margin: 0 auto;
  width: 300px;
  padding: 5px;
  border: 1px solid #cccccc;
`;

export default Card;
