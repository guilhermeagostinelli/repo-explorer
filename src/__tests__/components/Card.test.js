import React from "react";
import { create } from "react-test-renderer";
import Card from "../../components/Card";
import "jest-styled-components";

describe("<Card />", () => {
  it("should match the snapshot", () => {
    const card = create(<Card />);
    expect(card.toJSON()).toMatchSnapshot();
  });
});
