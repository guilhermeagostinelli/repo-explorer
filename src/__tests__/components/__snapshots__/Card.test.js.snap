// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> should match the snapshot 1`] = `
.c0 {
  margin: 0 auto;
  width: 300px;
  padding: 5px;
  border: 1px solid #cccccc;
}

<div
  className="c0"
/>
`;
